packages/xrpc-server/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@adxp/xrpc-server",
```

Comment:
make sure to add both of these pkgs to the root `tsconfig.json`

Reply:
Done